tests/configurable_cluster.py

```diff
@@ -140,6 +149,11 @@ def get_down_hostname(self, host_name):
 
     def exec_cmd_on_host(self, host, cmd, user=None, raise_error=True,
                          tty=False):
+        # Ensure command is invoked with sudo since EMR's login user is ec2-user
+        # If sudo is already present in the command then no error will occur
+        # as arbitrary nesting of sudo is allowed
```

Comment:
The first command you suggested works but the second doesn't. However, the second doesn't even if the command is executed without sudo so I'm not introducing any regressions and I won't debug further.

As for the point of invoking sudo for everything what I'll do is make sudo configurable so that when we start the "no sudo" work it can be turned off.